backend/Squirrel.Core/Squirrel.Core.BLL/Services/ProjectService.cs

```diff
@@ -0,0 +1,86 @@
+﻿using AutoMapper;
+using Microsoft.EntityFrameworkCore;
+using Squirrel.Core.Common.DTO.Projects;
+using Squirrel.Core.BLL.Interfaces;
+using Squirrel.Core.BLL.Services.Abstract;
+using Squirrel.Core.Common.Enums;
+using Squirrel.Core.DAL.Context;
+using Squirrel.Core.DAL.Entities;
+
+namespace Squirrel.Core.BLL.Services
+{
+    public class ProjectService : BaseService, IProjectService
+    {
+        public ProjectService(SquirrelCoreContext context, IMapper mapper): base(context, mapper){ }
+
+        public async Task<ProjectDto> AddProjectAsync(ProjectDto projectDto)
+        {
+            var projectEntity = _mapper.Map<Project>(projectDto);
+            
+            projectEntity.Engine = (int)projectDto.Engine;
+            
+            _context.Projects.Add(projectEntity);
```

Comment:
There is AddAsync method, generally it is better to use async methods to avoid thread blocking